platform/.bazelrc

```diff
@@ -1 +1,2 @@
+startup --host_jvm_args=-Xmx4G
```

Comment:
is this the smallest order of magnitude which worked?

Reply:
No, this was just a number I dumped into it. Do you want me to fiddle with it?

Reply:
No, should be fine because JVM doesn't commit this amount of memory